<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KMFJ2QA" ignored="0" name="Sanity7" tests="1" failures="1" timestamp="27 Mar 2019 16:15:46 GMT" time="60.611" errors="0">
  <testcase name="LoginTest" time="11.085" classname="Sanity.LoginTest7">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Sanity.LoginTest7.LoginTest(LoginTest7.java:26)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Sanity7 -->
